src/security/credentials.rs

```diff
@@ -170,18 +289,27 @@ impl Drop for ServerCredentialsBuilder {
 ///
 /// Use [`ServerCredentialsBuilder`] to build a [`ServerCredentials`].
 pub struct ServerCredentials {
-    creds: *mut grpc_server_credentials,
+    c_creds: *mut grpc_server_credentials,
+    cert_user_data: Option<Box<CertUserData>>,
```

Comment:
`UserFetcher` is not a part of `ServerCredentials`, it produces credentials. It' not appropriate to put it here.

Reply:
Already put it into `ServerBuilder` and `Server`.